profiler/src/main/java/com/splunk/opentelemetry/profiler/ThreadDumpRegion.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+public class ThreadDumpRegion {
+  public final String threadDump;
+  public int startIndex;
+  public int endIndex;
+
+  public ThreadDumpRegion(String threadDump) {
```

Comment:
Is this just for tests? It's a pretty simple constructor, can we inline it and not keep it in the main sources?

Reply:
Removed.